controllers/changetierrequest/changetierrequest_controller.go

```diff
@@ -178,6 +178,7 @@ func (r *Reconciler) changeTier(logger logr.Logger, changeTierRequest *toolchain
 			mur.Spec.UserAccounts[i].Spec.NSTemplateSet = newNsTemplateSet
 		}
 	}
+	mur.Spec.TierName = changeTierRequest.Spec.TierName
```

Comment:
shouldn't this have been:
```suggestion
	if mur.Spec.TierName != changeTierRequest.Spec.TierName {
		mur.Spec.TierName = changeTierRequest.Spec.TierName
		changed = true
	}
```